api/src/main/java/com/velocitypowered/api/event/Async.java

```diff
@@ -0,0 +1,17 @@
+package com.velocitypowered.api.event;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Marks an event handler method as async. By default are events handled on the
+ * network task executor, using this annotations performs them on an asynchronous
+ * task executor.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface Async {
```

Comment:
`AsyncSubscriber` or something, maybe?

Reply:
In the code prototype he showed off on Discord, `@Async` is meant to be combined with `@Subscribe`.

Reply:
Sure, but `Async` is a bit vague here, especially when working with certain other libraries that have classes named the same.

Reply:
I don't think `@Async` will be a big problem, you're more likely to confuse the guava `Subscribe` with the velocity one with the current libraries, I don't see another `Async` class or annotation

Reply:
Perhaps then Velocity's annotation should just get a new `async` boolean property?

Reply:
Personally I prefer `@Async`, but an `async` property is also an option. @astei what do you prefer?

Reply:
`async` property gets my vote.

Reply:
`async` property